src/main/java/gov/nasa/pds/registry/common/cfg/ConfigManager.java

```diff
@@ -0,0 +1,92 @@
+package gov.nasa.pds.registry.common.cfg;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import jakarta.xml.bind.JAXBContext;
+import jakarta.xml.bind.JAXBException;
+import org.glassfish.jaxb.runtime.v2.JAXBContextFactory;
+import gov.nasa.pds.registry.common.ConnectionFactory;
+import gov.nasa.pds.registry.common.EstablishConnectionFactory;
+import gov.nasa.pds.registry.common.meta.cfg.FileRefRule;
+
+/**
+ * Harvest configuration file reader.
+ * 
+ * @author karpenko
+ */
+public class ConfigManager
+{
+  private static HashMap<String,String> indexNodeMap = new HashMap<String,String>();
+  static {
+    indexNodeMap.put("pds-atm-registry", "PDS_ATM");
```

Comment:
Hi @al-niessner , @jordanpadams , I wanted to merge that PR, but I notice that piece of code which is outdated compared to the fix made in https://github.com/NASA-PDS/harvest/pull/188/files

Reply:
@tloubrieu-jpl 

Git will do the right thing or it is supposed to anyway. It looks like there is no conflict with main, so it should resolve correctly. I guess if you forced a rebase  or something where it thinks these edits are off the new HEAD of main then you need to just throw away this work because none of the merging will go well. I doubt you did that though.